src/main/java/bio/terra/cli/command/helperclasses/options/Format.java

```diff
@@ -78,7 +78,7 @@
    */
   public static <T> void printJson(T returnValue) {
     // use Jackson to map the object to a JSON-formatted text block
-    ObjectMapper objectMapper = new ObjectMapper();
+    ObjectMapper objectMapper = new ObjectMapper().findAndRegisterModules();
```

Comment:
I was thinking of this too...  If we do that we'll need to be careful to remember that any of the features we attach to the static instance are then sticky (as in `readFileIntoJavaObject`)...  But I'm thinking that with the CLI being short lived, this will likely only happen once per process lifecycle anyway?

Is having to call `findAndRegisterModules()` a consequence of using the newer version of Jackson?

Reply:
It doesn't seem noticeably slower to me, but I didn't benchmark or try to measure the speed in any way.

We could re-use a single `ObjectMapper` object, and then create a new one if a caller wants to attach features? That seems like a good improvement to me.

No, `findAndRegisterModules` is needed in order to decode Java date/time related classes (in this case `LocalDate`), because the logic that does so is in a separate library.

Reply:
Added a `JacksonMapper` utility class that maintains a singleton instance of the default `ObjectMapper`. If any mapper features are specified, we create a copy of the default object, so that these features are not sticky.